golden/golden.go

```diff
@@ -26,11 +26,17 @@ func Get(t require.TestingT, filename string) []byte {
 
 // Path returns the full path to a golden file
 func Path(filename string) string {
+	if filepath.IsAbs(filename) {
+		return filename
+	}
 	return filepath.Join("testdata", filename)
 }
 
 func update(t require.TestingT, filename string, actual []byte) {
 	if *flagUpdate {
+		if filepath.IsAbs(filename) {
+			return
+		}
```

Comment:
What's the rational for not updating the file when the path is absolute?

Reply:
Not much rational, I personally would just feel scared if it could update non testdata files as well. What are your thoughts on this? Worst case is probably when you test fixtures are not in git. (Uncommon for sure)

Reply:
I think it should be ok. It does require someone to use the flag, so just running `go test` won't accidentally update anything. I think it would be good to keep the behaviour consistent.

@vdemeester what do you think?